AppCenterCrashes/AppCenterCrashesTests/MSACErrorLogFormatterTests.mm

```diff
@@ -97,7 +97,6 @@ - (void)testCreateErrorReport {
   XCTAssertEqual([errorReport.appErrorTime timeIntervalSince1970], [crashReport.systemInfo.timestamp timeIntervalSince1970] + 0.999);
   assertThat(errorReport.appStartTime, equalTo(crashReport.processInfo.processStartTime));
 
-  XCTAssertEqualObjects(errorReport.device, self.deviceMock);
```

Comment:
Is it worth writing a function that would check the devices for being partly equal?
I guess you removed these lines because device is not equal now since it takes some data from crash report. But half of the object still should be equal.